build.gradle.kts

```diff
@@ -1,49 +1,52 @@
-import org.jetbrains.changelog.markdownToHTML
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 fun properties(key: String) = project.findProperty(key).toString()
 
 plugins {
+    val kotlinVersion = "1.5.10"
+    // Spring
+    id("io.spring.dependency-management") version "1.0.9.RELEASE"
+    id("org.springframework.boot") version "2.3.0.RELEASE" apply false
+    // Docker
+    id("com.palantir.docker") version "0.25.0" apply false
     // Java support
     id("java")
     // Kotlin support
-    id("org.jetbrains.kotlin.jvm") version "1.5.10"
+    id("org.jetbrains.kotlin.jvm") version kotlinVersion
     // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
     id("org.jetbrains.intellij") version "1.1.3"
     // gradle-changelog-plugin - read more: https://github.com/JetBrains/gradle-changelog-plugin
     id("org.jetbrains.changelog") version "1.1.2"
+
+    kotlin("plugin.spring") version kotlinVersion
 }
 
 group = properties("pluginGroup")
 version = properties("pluginVersion")
 
-// Configure project's dependencies
-repositories {
-    mavenCentral()
+allprojects {
+    group = group
 }
 
-//dependencies {
-//    implementation("io.github.microutils:kotlin-logging:1.12.5") { exclude("org.slf4j") }
-//}
-
-// Configure gradle-intellij-plugin plugin.
-// Read more: https://github.com/JetBrains/gradle-intellij-plugin
-intellij {
-    pluginName.set(properties("pluginName"))
-    version.set(properties("platformVersion"))
-    type.set(properties("platformType"))
-    downloadSources.set(properties("platformDownloadSources").toBoolean())
-    updateSinceUntilBuild.set(true)
+subprojects {
+    apply(plugin = "base")
+    apply(plugin = "io.spring.dependency-management")
 
-    // Plugin Dependencies. Uses `platformPlugins` property from the gradle.properties file.
-    plugins.set(properties("platformPlugins").split(',').map(String::trim).filter(String::isNotEmpty))
-}
+    // Configure project's dependencies
+    repositories {
+        mavenCentral()
+        jcenter()
+    }
 
-// Configure gradle-changelog-plugin plugin.
-// Read more: https://github.com/JetBrains/gradle-changelog-plugin
-changelog {
-    version = properties("pluginVersion")
-    groups = emptyList()
+    dependencyManagement {
+        dependencies {
+            dependency("io.github.microutils:kotlin-logging:1.7.8")
+            dependency("com.squareup.okhttp3:okhttp:4.9.1")
+            dependency("com.google.code.gson:gson:2.8.6")
+            dependency("com.fasterxml.jackson.module:jackson-module-kotlin:2.13.0-rc1")
```

Comment:
Dit is een release candidate, dus die zou ik niet gebruiken. Beter terug naar 2.12.5